manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -315,10 +315,13 @@ Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
                 ,groupField: this.config.groupBy || 'name'
                 ,storeId: this.config.storeId || Ext.id()
                 ,autoDestroy: true
-				,listeners:{
+                ,listeners:{
                     load: function(){
-						Ext.getCmp('modx-content').doLayout(); /* Fix layout bug with absolute positioning */
-					}
+                        cmp = Ext.getCmp('modx-content');
```

Comment:
Bit nitpicking, but this should be `var cmp = ...`; it's implicitly creating the variable now.